### PR TITLE
fix(style,frontend): Print terminal output character by character #187

### PR DESCRIPTION
**Pull request by Yashwanth-Chandrakumar**
Added a 50ms delay between printing of commands in OpenDevin's Terminal

### CODE REVIEW

**Comment by 808vita** [comment]
Hi,
I noticed that you removed some comments in your pull request. I believe that the comments you removed were necessary or helpful. I'm wondering why you did that.

**Comment by Yashwanth-Chandrakumar** [comment]
Sorry for that issue i have added the comments again.

File: frontend/src/components/Terminal.tsx [hunk 1]
```diff
@@ -45,10 +53,8 @@ type TerminalProps = {
   hidden: boolean;
 };
 
-/**
- * The terminal's content is set by write messages. To avoid complicated state logic,
- * we keep the terminal persistently open as a child of <App /> and hidden when not in use.
- */
+// * The terminal's content is set by write messages. To avoid complicated state logic,
+// * we keep the terminal persistently open as a child of <App /> and hidden when not in use.
```
Comment (rbren) [inline frontend/src/components/Terminal.tsx:57]: Can you revert this change?

Comment (Yashwanth-Chandrakumar) [reply frontend/src/components/Terminal.tsx:57]: Dont we need those comments

**Comment by Yashwanth-Chandrakumar** [comment]
Hi @rbren i have a small doubt when i create a pull request and commit to the branch in my forked repo, why does it show like 9 commits ahead of OpenDevin/Opendevin:main

**Comment by rbren** [comment]
OK I think I miscommunicated what I want to see here 😄 

I want it to look like the AI is typing live, like it does with ChatGPT
typing

**Comment by rbren** [comment]
> i have a small doubt when i create a pull request here and commit to the branch in my forked repo, why does it show like 9 commits ahead of OpenDevin/Opendevin:main

This means that you've made 9 new commits that aren't merged into the `main` branch--it's expected

**Comment by Yashwanth-Chandrakumar** [comment]
> Would be good to have this behavior for both chat messages and for the terminal

Ok Sure

**Comment by rbren** [comment]
> Ok thanks for the comfirmation so when you merge my pull request the 9 commits head of would disappear right

yup exactly
